### PR TITLE
[METIS] Trigger rebuild

### PR DESCRIPTION
**Pull request by giordano**
Let's see if this fixes FreeBSD craziness.

### CODE REVIEW

File: M/METIS/build_tarballs.jl [hunk 1]
```diff
@@ -29,6 +29,10 @@ cmake $WORKSPACE/srcdir/metis-5.1.0/ \
     -DGKLIB_PATH=$WORKSPACE/srcdir/metis-5.1.0/GKlib \
     -DSHARED=1
 make -j${nproc} install
+
+if [[ "${target}" == *-freebsd* ]]; then
+    file ${libdir}/libmetis.so
```
Comment (ararslan) [inline M/METIS/build_tarballs.jl]: Maybe throw in an `ldd` and even a splash of `objdump -p` for good measure?

**Comment by giordano** [comment]
Ok, hopefully this time we'll get a proper FreeBSD library.  From the debug build:
```
[18:40:24]  ---> file ${libdir}/libmetis.so
[18:40:24] /workspace/destdir/lib/libmetis.so: ELF 64-bit LSB shared object, x86-64, version 1 (FreeBSD), dynamically linked, not stripped
[18:40:24]  ---> ldd ${libdir}/libmetis.so
[18:40:25] /workspace/destdir/lib/libmetis.so: error while loading shared libraries: /workspace/destdir/lib/libmetis.so: ELF file OS ABI invalid
[18:40:25]   SYMENT               0x0000000000000018
[18:40:25]   PLTGOT               0x000000000027a310
[18:40:25]   PLTRELSZ             0x00000000000021a8
[18:40:25]   PLTREL               0x0000000000000007
[18:40:25]   JMPREL               0x000000000000d248
[18:40:25]   RELA                 0x000000000000ceb8
[18:40:25]   RELASZ               0x0000000000000390
[18:40:25]   RELAENT              0x0000000000000018
[18:40:25]   VERNEED              0x000000000000ce58
[18:40:25]   VERNEEDNUM           0x0000000000000002
[18:40:25]   VERSYM               0x000000000000c708
[18:40:25]   RELACOUNT            0x000000000000000e
[18:40:25] 
[18:40:25] Version References:
[18:40:25]   required from libm.so.5:
[18:40:25]     0x077a28b0 0x00 03 FBSD_1.0
[18:40:25]   required from libc.so.7:
[18:40:25]     0x077a28b3 0x00 05 FBSD_1.3
[18:40:25]     0x077a28b1 0x00 04 FBSD_1.1
[18:40:25]     0x077a28b0 0x00 02 FBSD_1.0
[18:40:25] 
[18:40:25]  ---> readelf -h ${libdir}/libmetis.so
[18:40:25] ELF Header:
[18:40:25]   Magic:   7f 45 4c 46 02 01 01 09 00 00 00 00 00 00 00 00 
[18:40:25]   Class:                             ELF64
[18:40:25]   Data:                              2's complement, little endian
[18:40:25]   Version:                           1 (current)
[18:40:25]   OS/ABI:                            UNIX - FreeBSD
[18:40:25]   ABI Version:                       0
[18:40:25]   Type:                              DYN (Shared object file)
[18:40:25]   Machine:                           Advanced Micro Devices X86-64
[18:40:25]   Version:                           0x1
[18:40:25]   Entry point address:               0x10a90
[18:40:25]   Start of program headers:          64 (bytes into file)
[18:40:25]   Start of section headers:          504048 (bytes into file)
[18:40:25]   Flags:                             0x0
[18:40:25]   Size of this header:               64 (bytes)
[18:40:25]   Size of program headers:           56 (bytes)
[18:40:25]   Number of program headers:         6
[18:40:25]   Size of section headers:           64 (bytes)
[18:40:25]   Number of section headers:         31
[18:40:25]   Section header string table index: 28
```
In particular, `OS/ABI` should finally be correct.

**Comment by ararslan** [comment]
> In particular, OS/ABI should finally be correct.

`ldd` says it isn't though?

**Comment by giordano** [comment]
The only `ldd` available in the environment is `/usr/bin/ldd`, so it's for Linux.  It's no surprise that it can't open a library for FreeBSD, in fact I protected the call with `|| true`.

**Pull request merged**